packages/fxa-content-server/tests/functional.js

```diff
@@ -6,20 +6,15 @@ const testsSettings = require('./functional_settings');
 module.exports = testsSettings.concat([
   'tests/functional/fx_browser_relier.js',
   'tests/functional/oauth_webchannel.js',
```

Comment:
Just curious, is there a reason why some tests are commented out and some are deleted?